openapi/v2/Bom/bom-get.yml

```diff
@@ -0,0 +1,209 @@
+openapi: 3.0.0
+info:
+  version: '3.0'
+  title: Get Bom API
+  license:
+    name: MIT
+servers:
+- url: https://{orgId}.salesforce.com/services/apexrest/{namespace}/v3
+  variables:
+    namespace:
+      default: PDLM
+      description: This value should default to PDLM for production environments.
+        For dev environments, provide the namespace to your dev org.
+paths:
+  '/item/{itemNumber}/bom':
+    get:
+      summary: BOM
+      description: Query to return a list of assembly records which represent the full BOM for the
+        given Item.
+      operationId: getBom
+      tags:
+      - Bill of Material
+      - BOM
+      - Assembly
+      parameters:
+      - name: itemNumber
+        in: path
+        description: Item Revision Id or Item Number of assembly to retrieve the BOM for. If Item Id or Item Number is provided, will default to latest released revision or latest revision if there is no released revision.
+        required: true
+        schema:
+          type: string
+          format: string
+      - name: level
+        in: query
+        description: Level of BOM to fetch. Defaults to All levels.
+        required: false
+        schema:
+          type: string
+          format: string
+          default: 'all'
```

Comment:
Looked at the code and were expecting to receive an Integer. Should we specify an number here is required and not string like 'first' or 'second'?
Also in package, we should handle scenario where user passes in 'all' as its default.
@pkjayk what do you think default 'all' or default '0' for all?

Reply:
I think using `all` is better. Can the package expect a string instead?

Reply:
@pkjayk So we want it to be able to accept either 'all' or '1', '2', '3', etc?